pkg/state/type.go

```diff
@@ -0,0 +1,19 @@
+package state
+
+type State struct {
+	Interfaces []interface{} `json:"interfaces"`
+	Routes     *Routes       `json:"routes,omitempty"`
+}
+
+type Routes struct {
+	Config  []interface{} `json:"config"`
+	Running []interface{} `json:"running"`
+}
+
+type InterfacesState struct {
+	Interfaces []Interface `json:"interfaces"`
+}
+
+type Interface struct {
+	Name string `json:"name"`
+}
```

Comment:
We don't need this to be public "yet"

Reply:
You mean the members or the structures?

Reply:
I guess you meant the struct. I changed it.